packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -61,6 +61,23 @@ export class HttpServer extends RunnableModule {
     return result;
   }
 
+  private checkReqVersion(version: string) {
+    return (req: express.Request, res: express.Response, next: express.NextFunction) => {
+      const [major, minor] = version.split('.');
+      const versionRegex = new RegExp(`^${major}\\.(\\d+)\\.(\\d+)`);
```

Comment:
nit: those could be a cached in outer scope. Also no need to capture 'patch' in a group

Reply:
done here: [c7bb59e](https://github.com/input-output-hk/cardano-js-sdk/pull/1022/commits/c7bb59ec2741ee019543e7aaffbea3e18bda4008)